tools/protoc-plugin/src/main/java/io/spine/tools/protoc/SpineProtoGenerator.java

```diff
@@ -159,6 +178,20 @@ private CodeGeneratorResponse process(TypeSet types) {
         return response;
     }
 
+    private Set<CompilerOutput> processTypes(TypeSet types) {
```

Comment:
Having two methods `CodeGeneratorResponse process(TypeSet types)` and `Set<CompilerOutput> processTypes(TypeSet types)` is confusing. They are both private and called very similar, but do different things. Consider renaming.